doc/releases/release-notes-3.1.rst

```diff
@@ -162,6 +221,15 @@ Drivers and Sensors
 
 * CAN
 
+  * Switched from transmitting CAN frames in FIFO/chronological order to transmitting according to
+    CAN-ID priority (NXP FlexCAN, ST STM32 bxCAN, Bosch M_CAN, Microchip MCP2515).
+  * Added support for ST STM32U5 to the ST STM32 bxCAN driver.
```

Comment:
bxCAN is STM32's classical CAN peripheral, only present in older MCUs. New ones like the STM32U series all got FDCAN (based on M_CAN).

```suggestion
  * Added support for ST STM32U5 to the ST STM32 FDCAN driver.
```

Reply:
@martinjaeger Fixed, thank you.